api/src/main/java/jakarta/servlet/http/HttpServlet.java

```diff
@@ -543,6 +602,9 @@ protected void service(HttpServletRequest req, HttpServletResponse resp) throws
             maybeSetLastModified(resp, lastModified);
             doHead(req, resp);
 
+        } else if (method.equals(METHOD_PATCH)) {
```

Comment:
Can you add this at the end of the if statement? The more common methods should be at the top.

Reply:
Fixed